internal/backup.go

```diff
@@ -322,3 +323,19 @@ func shouldUnwrapTar(tarName string, sentinelDto BackupSentinelDto, filesToUnwra
 
 	return false
 }
+
+func getLastWalFilename(backup *Backup) (string, error) {
+	meta, err := backup.fetchMeta()
+	if err != nil {
+		tracelog.DebugLogger.FatalError(err)
+		return "", err
+	}
+	timelineID64, err := strconv.ParseUint(backup.Name[len(utility.BackupNamePrefix):len(utility.BackupNamePrefix)+8], 0x10, sizeofInt32bits)
```

Comment:
Extract to variable value that you use more than one time. len(utility.BackupNamePrefix)